test/Test/Lecture4.hs

```diff
@@ -90,7 +90,7 @@ lecture4Spec = describe "Lecture 4" $ do
             }
 
     let stringStats =
-            [ "Total positions:       : 3"
+            [ "Total positions        : 3"
```

Comment:
Oops, this looks like a typo in tests 👀 

Feel free to submit the fix to this test case by opening a PR directly to the course 👍🏻 
I would appreciate that a lot 🙏🏻